src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -190,13 +203,15 @@ public void stop() {
     }
 
     public void coast() {
-        driveMotor.setIdleMode(IdleMode.kCoast);
-        turningMotor.setIdleMode(IdleMode.kCoast);
+        int errorCode = driveMotor.setIdleMode(IdleMode.kCoast).value;
+        errorCode += turningMotor.setIdleMode(IdleMode.kCoast).value;
+        if(errorCode != 0){DriverStation.reportError("An Error has occured in SwerveModule.coast() Code:" + REVLibError.fromInt(errorCode).toString(), null);}
```

Comment:
you can't translate the error code to a string if you add them together before hand.

Reply:
should be resolved by doing them individually, not 100% sure though